scss/container.scss

```diff
@@ -0,0 +1,64 @@
+@import "helpers";
+
+
+@function get-container-name($data) {
```

Comment:
Please put a comment for this function including an example to know what the data looks like before and after using this function. Something like:

```scss
// Get container name
// "foo" => "bar"
```